.github/workflows/tests.yml

```diff
@@ -103,8 +103,6 @@ jobs:
           brew tap homebrew/portable-ruby
           brew tap homebrew/services
 
-          brew update-reset Library/Taps/homebrew/homebrew-bundle
```

Comment:
I've now removed it. I can put it back with the correct path, but given that this has been a silent no-op for a while now I suspect we don't need it.

Reply:
@carlocab Yup, all good, thanks!